ci/pipeline.yml

```diff
@@ -29,10 +29,20 @@ jobs:
 
       - task: build-linux
         file: davcli-src/ci/tasks/build.yml
+        params:
+          GOOS: linux
 
       - put: release-bucket-linux
         params: {file: out/davcli-*-linux-amd64}
 
+      - task: build-windows
```

Comment:
Let's parallelize this. Perhaps one parallel job for building, and a second one for uploading (just in case one platform build fails we then avoid uploading half the artifacts for the version and confusing the world).